include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -76,6 +76,13 @@ ERROR(error_unknown_arg,none,
       "unknown argument: '%0'", (StringRef))
 ERROR(error_invalid_arg_value,none,
       "invalid value '%1' in '%0'", (StringRef, StringRef))
+ERROR(error_invalid_locale_code,none,
+      "unsupported locale code; supported locale codes are: '%0'", (StringRef))
+WARNING(locale_path_not_found,none,
```

Comment:
could you please prefix this and other new warning with `warn_` to avoid confusion in the future?